ada_feeding/config/ada_feeding_action_servers_current.yaml

```diff
@@ -0,0 +1,6 @@
+# This file is auto-generated by create_action_servers.py
```

Comment:
Now that this is in a commit and is updated by code, it should be added to `.gitignore`.

Reply:
I initially put it in the `.gitignore`, but then realized that `.gitignore` doesn't do anything for files that are already in the repository. It seems like (based on this thread) the only thing we can do is running `git update-index --assume-unchanged ada_feeding/config/ada_feeding_action_servers_current.yaml` in our local copy, but we can't do any global configurations to ignore changes to this file.

I'd say we just have to be vigilant in PRs to verify changes to this file are not pushed.

Reply:
Actually, I think `update-index --assume-unchanged` can mess up some git checks; like it thought I had local changes in the current.yaml file even though I didn't, so prevented me from changing branches. I'd say we should not use that command, and instead just be vigilant on the PR side to verify changes to this file don't get passed in.